SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -39,6 +83,27 @@ void UGlobalStateManager::ApplyUpdate(const Worker_ComponentUpdate& Update)
 	}
 }
 
+void UGlobalStateManager::ApplyMapUpdate(const Worker_ComponentUpdate& Update)
+{
+	Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(Update.schema_type);
+
+	if (Schema_GetObjectCount(ComponentObject, 1) == 1)
```

Comment:
check me up

Reply:
With component updates it's common to not update all components in an entity. These if's mean we only update a component if it's present in the `Worker_ComponentUpdate`. Checking would fail badly here.